packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -3,6 +3,7 @@ import express, { Request, Response, NextFunction } from 'express'
 import Ceramic from '@ceramicnetwork/ceramic-core'
 import type { CeramicConfig } from "@ceramicnetwork/ceramic-core";
 import { DoctypeUtils, RootLogger, Logger } from "@ceramicnetwork/ceramic-common"
+import {loggerProviderPlugins} from "./ceramic-plugins"
```

Comment:
Naming it to `ceramic-plugins` seems to generic? Isn't it more of a custom logger that is passed as an option to the ceramic instance?

Reply:
Yeah I was thinking maybe `ceramic-logger-plugins` is better